libs/application/templates/accident-notification/src/utils/index.ts

```diff
@@ -35,7 +35,11 @@ export const getAttachmentTitles = (answers: AccidentNotification) => {
 
   if (hasAttachment(deathCertificateFile))
     files.push(attachments.documentNames.deathCertificate)
-  if (hasAttachment(injuryCertificateFile))
+  if (
+    hasAttachment(injuryCertificateFile) &&
+    answers.injuryCertificate?.answer !==
+      AttachmentsEnum.HOSPITALSENDSCERTIFICATE
```

Comment:
Please use getValueViaPath to get the answer, theres a big refactor that fixes the other util functions. Apart from that it looks good.